packages/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.0.3+1
```

Comment:
nit: space before version and after changelog notes

Reply:
done